src/components/Confirmation Modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,35 @@
+import { Modal, Button } from "@mantine/core"
+import styles from "./ConfirmationModal.module.css"
+
+interface ConfirmationModalProps {
+  openedModal: boolean
+  close: () => void
+  onClearChat: () => void
+  modalMessage: string
+}
+
+const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
```

Comment:
```suggestion
const ConfirmationModal: ConfirmationModalProps = ({
```
Can we use this type for consistency instead of React.FC?